src/panels/lovelace/common/generate-lovelace-config.ts

```diff
@@ -7,6 +7,7 @@ import splitByGroups from "../../../common/entity/split_by_groups";
 import computeObjectId from "../../../common/entity/compute_object_id";
 import computeStateDomain from "../../../common/entity/compute_state_domain";
 import { LocalizeFunc } from "../../../mixins/localize-base-mixin";
+import computeDomain from "../../../common/entity/compute_domain";
 
 interface CardConfig {
```

Comment:
We should use [LovelaceConfig](https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/panels/lovelace/types.ts#L3) (Should be LovelaceCardConfig) and move the rest of the interfaces to types so it can be reused?

Reply:
In a new pr but yes. I didn't want to rename the other one , impacts so many files